### ISSUE TITLE
mockup contracts

### ISSUE THREAD

**Issue by Licenser** [Licenser/orgtool2#6]
@skoma Is this still a thing?

**Comment by skoma** [comment]
Yes, still a thing. The idea is to add something like http://www.destinylfg.net/

Proposed LFG Features
- Communicate Contract creation with #Contracts channel this includes
      - Creation of New contracts
      - Currently Active Contracts
      - Contract member slot status
- Communicate with Contract Creator to:
      - Let them know a contract has been created
      - Update them when new contractees join and provide them with inform them of who they are
      - Provide a status update closer to the time to inform them that a contract is about to commence
-Communicate with Contractees (Signups)
      - Provide a status update  closer to the time to inform them that a contract is about to  commence
      - Inform them if the contract gets cancelled / changes

Discord features
-Allow members to sign up for a role via -Subscribe [role]

**Comment by Licenser** [comment]
hmm LFG and contracts were something different in my mind. Contracts seem more inter org, while LFG seems more in the same org? Perhaps we sit this one out a bit until we get a basic idea what posibilites CIG will offer for us to work with?
